cilium-cli/connectivity/builder/pod_to_pod_encryption.go

```diff
@@ -27,6 +28,14 @@ func (t podToPodEncryption) build(ct *check.ConnectivityTest, _ map[string]strin
 
 	newTest("pod-to-pod-with-l7-policy-encryption", ct).
 		WithCondition(func() bool { return !ct.Params().SingleNode }).
+		WithCondition(func() bool {
+			if ok, _ := ct.Features.MatchRequirements(features.RequireMode(features.EncryptionPod, "ipsec")); ok {
+				if !versioncheck.MustCompile(">=1.15.0")(ct.CiliumVersion) {
+					return false
+				}
+			}
```

Comment:
Note that this would need to test for the specific `1.15.x` / `1.16.y` patch versions that include the fix.

Reply:
Ty Julian! Looking at the next [tags](https://github.com/cilium/cilium/tags), I should've just updated the test accordingly (next tags will be v1.15.11 and v1.16.4).